pyquil/tests/test_reference_density_simulator.py

```diff
@@ -255,3 +266,69 @@ def test_multiqubit_decay_bellstate():
     qam.execute(program)
 
     assert np.allclose(qam.wf_simulator.density, state)
+
+
+def test_for_negative_probabilities():
+    def _state_tomo_settings(qubits):
+        """Yield settings over itertools.product(I, X, Y, Z).
+
+        Used as a helper function to generate a state tomography experiment.
+
+        :param qubits: The qubits to tomographize.
+        """
+        n_qubits = len(qubits)
+        for o_ops in itertools.product([sI, sX, sY, sZ], repeat=n_qubits):
+            o_op = functools.reduce(mul, (op(q) for op, q in zip(o_ops, qubits)), sI())
+
+            yield ExperimentSetting(
+                in_state=zeros_state(qubits),
+                out_operator=o_op,
+            )
+
+    # trivial program to do state tomography on
+    prog = Program()
+    prog += I(0)
```

Comment:
Change to the one-liner `prog = Program(I(0))`

Reply:
- [ ] todo